### PR TITLE
docs: Review Create New Section doc 

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- Update `Create New Section` to work with new version.

## How to test it?

- You should be able to create a new section/component in the CMS following the [guide](https://faststore-site-rj6ydz4y8-faststore.vercel.app/docs/building-sections/creating-a-new-section).

### CODE REVIEW

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 1]
```diff
@@ -106,15 +129,16 @@ This section will receive the `link` and `title` defined in the sections.json fi
...
 ```
 
 ### Step 3: Synchronizing the new section with the Headless CMS
 
-1. In the terminal, run `yarn cms-sync`. This command will synchronize the new section you created with the @faststore/core.
+1. In the terminal, run `faststore dev`.
+2. Then, run `faststore cms-sync`. This command will synchronize the new section you created with the CMS.
 2. Go to the VTEX Admin and access **Storefront > Headless CMS**.
 3. Click on the `Home` content type.
 4. In the `Sections` tab, click the `+`, search for the new `Call to Action` section, and add it to your page.
```
Comment (hellofanny) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:144]: Add image here.

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 2]
```diff
@@ -15,85 +15,108 @@ import { Callout } from 'nextra-theme-docs'
   This documentation is currently under development.
 </Callout>
 
-In this guide, you'll learn how to create a new section for your store and making it available in the Headless CMS. This solution is useful when your store requires a section that is not available natively in FastStore.
+In this guide, you'll learn how to create a new section for your store and make it available in the VTEX Headless CMS. This solution is useful when your store requires a section not natively available in FastStore.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:18]:
```suggestion
In this guide, you'll learn how to create a new section for your store and make it available in the [VTEX Headless CMS](/docs/headless-cms-overview). This solution is useful when your store requires a section not natively available in FastStore.
```

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 3]
```diff
@@ -15,85 +15,108 @@ import { Callout } from 'nextra-theme-docs'
...
-**1. Integrate the store with the Headless CMS**
+**1. Integrate the store with the VTEX Headless CMS**
 
 All sections must be available in the Headless CMS so they can be added and managed on your store's pages. To integrate your FastStore project, please refer to the [VTEX Headless CMS integration track](/docs/headless-cms-integration/1-configuring-the-vtex-account).
 
 **2. Knowledge on how sections and content types work on Headless CMS**
 
-During the creation of a new section, you will create files such as `sections.json` and `content-types.json`, which follow a structure established for the Headless CMS. For more information about these files, refer to
-[Adding Content Types and Sections to the VTEX Headless CMS](/tbd)
+During the creation of a new section, you will create files such as `sections.json` and `content-types.json`, which follow a structure established for the Headless CMS.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:37]:
```suggestion
During the creation of a new section, you will create files such as `sections.json` and `content-types.json`, which follow a structure established for the VTEX Headless CMS.
```

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 4]
```diff
@@ -15,85 +15,108 @@ import { Callout } from 'nextra-theme-docs'
...
+          }
         }
       }
     }
   }
 ]
 ```
 
-This new section receives a title and a link that can points to some place in the store.
+This new section receives a title and a link pointing to somewhere.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:89]:
```suggestion
<Callout type="info" emoji="ℹ️">
  This new section receives a title and a link pointing to a specific location.
</Callout>
```

**Review by Mariana-Caetano — COMMENTED** [review]
LGTM 🚀
I suggested minor changes.

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 5]
```diff
@@ -106,15 +129,21 @@ This section will receive the `link` and `title` defined in the sections.json fi
 
 Open the index.tsx file and add the following code:
 
-```tsx
+```tsx filename="components/index.tsx"
```
Comment (hellofanny) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:132]:
```suggestion
```tsx filename="components/index.ts"
```

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 6]
```diff
@@ -106,15 +129,21 @@ This section will receive the `link` and `title` defined in the sections.json fi
 
 Open the index.tsx file and add the following code:
```
Comment (hellofanny) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:130]:
```suggestion
Open the `index.ts` file and add the following code:
```

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 7]
```diff
@@ -15,85 +15,108 @@ import { Callout } from 'nextra-theme-docs'
...
+      <a href={props.link.url}>{props.link.text}</a>
+    </section>
+  );
 }
 ```
 
-This section will receive the `link` and `title` defined in the sections.json file.
+This section will receive the `link` and `title` defined previously in the `sections.json` file.
 
 3. Create a file named `index.tsx` inside the components folder.
```
Comment (hellofanny) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:121]:
```suggestion
3. Create a file named `index.ts` inside the components folder.
```

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 8]
```diff
@@ -15,85 +15,108 @@ import { Callout } from 'nextra-theme-docs'
...
-To render the `CallToAction` section that you created in the previous step, you need to create the section.
+To render the `CallToAction` section you created in the previous step, you need to create this section component.
 
-1. If you don't already have it, create a folder named `components` inside the src directory.
+1. If you don't already have it, create a folder named `components` inside the `src` directory.
 2. Inside the `components` folder, create a file named `CallToAction.tsx` and add the following code:
 
-```tsx
-import React, { FunctionComponent } from 'react'
+```tsx filename="components/CallToAction.tsx"
```
Comment (hellofanny) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:98]:
```suggestion
```tsx filename="src/components/CallToAction.tsx"
```

**Review by eduardoformiga — APPROVED** [review]

**Comment by hellofanny** [comment]
I merged because a few important updates should go live ASAP! We can improve this doc later! :)

**Pull request merged**